### PR TITLE
Update caption to be more descriptive

### PR DESCRIPTION
**Pull request by lakamson**
Per request, making changes to caption description of slide deck!

Fixes
Closes
Resolves #2

### CODE REVIEW

File: _slides/13-lakamson.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to my ring!"
 ---
 
 Caption here
```
Comment (allthedoll) [inline _slides/13-lakamson.md]: Change the caption here, not in the title. 😉

File: _slides/13-lakamson.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to my ring!"
```
Comment (allthedoll) [inline _slides/13-lakamson.md]:
```suggestion
title: "Welcome to our slide deck!"
```

**Review by allthedoll — CHANGES_REQUESTED** [review]
Go ahead and put the caption in line 6!

**Comment by allthedoll** [comment]
We can continue to work in this pull request.

**Comment by ernest-phillips** [comment]
Sorry. closed this by mistake 🙇🏾

**Review by allthedoll — APPROVED** [review]

**Comment by lakamson** [comment]
👍🏾 Thanks @allthedoll

**Pull request merged**